components/Dropdown/Dropdown.js

```diff
@@ -0,0 +1,20 @@
+class Dropdown {
+    constructor(element, hideClass) {
+      this.element = element, // 1st step w js element functions 
```

Comment:
Love the comments you have as you write your code. I did this for my previous code and found them to be so helpful when it came time for the Sprint Challenge!